packages/mgt-spfx/.editorconfig

```diff
@@ -0,0 +1,25 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
Should we remove this to ensure there are no conflicting rules with our prettier config?